flink-doris-connector/src/main/java/org/apache/doris/flink/tools/cdc/mongodb/MongoDBDatabaseSync.java

```diff
@@ -181,13 +178,13 @@ public DataStreamSource<String> buildCdcSource(StreamExecutionEnvironment env) {
 
         String startupMode = config.get(SourceOptions.SCAN_STARTUP_MODE);
         switch (startupMode.toLowerCase()) {
-            case INITIAL_MODE:
+            case DatabaseSyncConfig.SCAN_STARTUP_MODE_VALUE_INITIAL:
                 mongoDBSourceBuilder.startupOptions(StartupOptions.initial());
                 break;
-            case LATEST_OFFSET_MODE:
+            case DatabaseSyncConfig.SCAN_STARTUP_MODE_VALUE_LATEST_OFFSET:
                 mongoDBSourceBuilder.startupOptions(StartupOptions.latest());
                 break;
-            case TIMESTAMP_MODE:
+            case DatabaseSyncConfig.SCAN_STARTUP_MODE_VALUE_TIMESTAMP:
```

Comment:
Can we use cdc variables directly here?

Reply:
These parameters of flink-cdc cannot be directly referenced because they are also privately defined.